test-support/ember-sinon-qunit/test.js

```diff
@@ -20,11 +20,21 @@ sinon.config = {
 };
 
 export default function test(testName, callback) {
+  // Enable adding to the context (i.e. `this` during the `test` function)
+  test.context = {};
```

Comment:
I like that even better -- I updated the PR. More interestingly, though, [all of this might be unnecessary altogether](https://github.com/elwayman02/ember-sinon-qunit/issues/25#issuecomment-220862823).